ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-rollback-nightly.yaml

```diff
@@ -0,0 +1,221 @@
+base_images:
+  ansible:
+    name: "4.17"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.17"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.17"
+    namespace: ocp
+    tag: hypershift-operator
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tools:
+    name: "4.17"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: 4.17.0-rc.0
```

Comment:
Hard code to rc.0, is it expected ?

Reply:
Yes. It's expected.

Reply:
What happens if rc.1, rc.2, ... comes out ?

Reply:
It will stick to rc.0 till ga.0 is available. The rule is that we always use the oldest stable version as the oldest rollback version in the testing.

Reply:
Ack